### ISSUE TITLE
Groups with nested schema displayed twice

### ISSUE THREAD

**Issue by TechplexEngineer** [aldeed/meteor-autoform#1681]
Fields with no `autoform: { group: "<>" }`  that are a child of another field that does have a "group" attr are displayed twice.

example schema:
```javascript
const nestedSchema = new SimpleSchema({
    field: {
        type: String
    }
});
const schema = new SimpleSchema({
    data: {
        type: nestedSchema,
        autoform: { group: "Group1" }
    }
});
```

Form will display:
```
Input for "field"
-- Group1 --
Data Panel
    input for "field"
```

If I'm not too far off the mark I think this function just needs to get a bit smarter:
https://github.com/aldeed/meteor-autoform/blob/devel/components/quickForm/quickForm.js#L132-L137

@SachaG was this intentional for a reason I can't think of?

**Comment by SachaG** [comment]
Sorry I don't really use Autoform anymore, can't say I remember much about that code…

**Comment by jankapunkt** [comment]
Could not reproduce. This issue may have been resolved with AutoForm 7.0.0, please migrate to the latest version and reopen in case the issue persist
